main_script.py

```diff
@@ -0,0 +1,125 @@
+from internal_functions.dna_rna_tools import transcribe
+from internal_functions.dna_rna_tools import reverse
+from internal_functions.dna_rna_tools import complement
+from internal_functions.dna_rna_tools import reverse_complement
```

Comment:
Так писать можно, это не ошибка, но лучше писать в более коротком виде
```suggestion
from internal_functions.dna_rna_tools import transcribe, reverse, complement, reverse_complement
```